source/exec.tex

```diff
@@ -1005,6 +1007,22 @@
 \tcode{forwarding_query(get_completion_scheduler<\exposid{completion-tag}>)}
 is a core constant expression and has value \tcode{true}.
 
+\rSec2[exec.get.await.adapt]{\tcode{execution::get_await_completion_adaptor}}
+
+\pnum
+\tcode{get_await_completion_adaptor} asks a queryable object for
+its associated awaitable completion adaptor.
+
+\pnum
+The name \tcode{get_await_completion_adaptor} denotes a query object.
```

Comment:
You can probably resolve the overfull hbox issues in this area by sprinkling some soft hyphens (`\-`) into these long identifiers.